packages/shared/components/IconBox.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
```

Comment:
I would rename this component to BoxedIcon and make it so its a component that uses Icon but wrapps it in a box, so you could use it like `<BoxedIcon classes="text-blue-200" boxClasses="bg-pink-100" icon={IconEnum.Lola} />`